### ISSUE TITLE
Unable to install latest version

### ISSUE THREAD

**Issue by pvorona** [nmccready/debug-fabulous#26]
```
#10 35.39 ➤ YN0000: │ youtube-dl-exec@npm:3.0.7 STDERR Error: Cannot find module '/app/node_modules/debug-fabulous/lib/index.js'. Please verify that the package.json has a valid "main" entry
#10 35.39 ➤ YN0000: │ youtube-dl-exec@npm:3.0.7 STDERR     at tryPackage (node:internal/modules/cjs/loader:443:19)
#10 35.39 ➤ YN0000: │ youtube-dl-exec@npm:3.0.7 STDERR     at Module._findPath (node:internal/modules/cjs/loader:711:18)
#10 35.39 ➤ YN0000: │ youtube-dl-exec@npm:3.0.7 STDERR     at Module._resolveFilename (node:internal/modules/cjs/loader:1126:27)
#10 35.39 ➤ YN0000: │ youtube-dl-exec@npm:3.0.7 STDERR     at Module._load (node:internal/modules/cjs/loader:981:27)
#10 35.39 ➤ YN0000: │ youtube-dl-exec@npm:3.0.7 STDERR     at Module.require (node:internal/modules/cjs/loader:1231:19)
#10 35.39 ➤ YN0000: │ youtube-dl-exec@npm:3.0.7 STDERR     at require (node:internal/modules/helpers:177:18)
#10 35.39 ➤ YN0000: │ youtube-dl-exec@npm:3.0.7 STDERR     at Object.<anonymous> (/app/node_modules/debug-logfmt/index.js:27:21)
#10 35.39 ➤ YN0000: │ youtube-dl-exec@npm:3.0.7 STDERR     at Module._compile (node:internal/modules/cjs/loader:1364:14)
#10 35.39 ➤ YN0000: │ youtube-dl-exec@npm:3.0.7 STDERR     at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)
#10 35.39 ➤ YN0000: │ youtube-dl-exec@npm:3.0.7 STDERR     at Module.load (node:internal/modules/cjs/loader:1203:32) {
#10 35.39 ➤ YN0000: │ youtube-dl-exec@npm:3.0.7 STDERR   code: 'MODULE_NOT_FOUND',
#10 35.39 ➤ YN0000: │ youtube-dl-exec@npm:3.0.7 STDERR   path: '/app/node_modules/debug-fabulous/package.json',
#10 35.39 ➤ YN0000: │ youtube-dl-exec@npm:3.0.7 STDERR   requestPath: 'debug-fabulous'
#10 35.39 ➤ YN0000: │ youtube-dl-exec@npm:3.0.7 STDERR }
```

**Comment by Kikobeats** [comment]
It seems transpiled files from TypeScript are not being published at npm registry .cc @nmccready

**Comment by nmccready** [comment]
Which version was giving you fits? Cause 2.0.6 seems to load when tested.

**Comment by nmccready** [comment]
NVM I am seeing it in 2.0.6 thanks

**Comment by nmccready** [comment]
Finally fixed it in 2.0.8 https://github.com/nmccready/debug-fabulous/actions/runs/10851335289/job/30114911657
